content/advanced/330_servicemesh_using_appmesh/canary_deployment/testing_v2.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Testing Canary"
+date: 2020-01-27T08:30:11-07:00
+weight: 100
+---
+
+Get the Loadbalancer endpoint
+```bash
+export LB_NAME=$(kubectl get svc ingress-gw -n prodcatalog-ns -o jsonpath="{.status.loadBalancer.ingress[*].hostname}") 
+echo $LB_NAME
+```
+
+Go to the above Load Balancer endpoint URL, you should see below screen which shows that Catalog Detail `proddetail-v1` Version 1 is being accessed.
```

Comment:
'Go to the above .. URL' maybe should be "now, back in your browser, note that version 1 is being used"... the refresh a few times. "Go to the above" means they might re-open it again, and it could start v2

Reply:
updated